src/script.js

```diff
@@ -182,6 +184,13 @@ export function getPartnerAttributionID() : ?string {
     return getSDKAttribute(SDK_SETTINGS.PARTNER_ATTRIBUTION_ID);
 }
 
+export function getPageType() : ?string {
+    const pageType = getSDKAttribute(SDK_SETTINGS.PAGE_TYPE, '');
+    const validPageType = pageType.length && PAGE_TYPES[pageType.toUpperCase()] !== undefined;
```

Comment:
Can we do `values(PAGE_TYPES).indexOf(pageType) !== -1` (`values` can be imported from `belter/src`).

If invalid, can we throw an error to let the merchant know

Reply:
Updated code and tests.